.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+	presets: ['latest'],
+	plugins: [
+		'transform-object-rest-spread',
+		['transform-runtime', {polyfill: false, regenerator: true}],
```

Comment:
polyfill: false reduces the size of the build a lot, but we need the regenerator runtime for the async/await compilation